File: swagger/models/get_resources_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetResourcesResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		resources
		Contains information about resources as the business.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Resources': 'resources',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'resources': 'Resources',
		}

		'PaginationResponse': ('pagination_response', PaginationResponse),
	resources = None

File: swagger/Models/GetResourcesResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetResourcesResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about resources as the business.
	 * 
	 * @var Resource[] 
	 */
	public $resources;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'Resources' => ['resources', 'array', Resource::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'resources' => ['Resources', 'array', Resource::class],
			];
	}
}
